docs/root/configuration/http/http_filters/kill_request_filter.rst

```diff
@@ -0,0 +1,39 @@
+.. _config_http_filters_kill_request:
+
+Kill Request
+===============
+
+The KillRequest filter can be used to crash Envoy when receiving a Kill request
+and measure the blast radius. The KillRequest filter is disabled at build time by default.
```

Comment:
drive by comment: what does blast radius mean in this context?

Reply:
Thanks Otto. Here the blast radius means when we send a kill request to 1 Envoy among a pool of Envoys, how many neighboring Envoy will be affected. This is specific to the downstream use case, so will remove this from the doc.